Source/ACE.DatLoader/Entity/AnimationHooks/ReplaceObjectHook.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace ACE.DatLoader.Entity.AnimationHooks
+{
+    public class ReplaceObjectHook : IHook
+    {
+        public AnimationPartChange APChange { get; set; }
+
+        public static ReplaceObjectHook ReadHookType(DatReader datReader)
+        {
+            ReplaceObjectHook r = new ReplaceObjectHook();
+
+            AnimationPartChange ao = new AnimationPartChange();
+            ao.PartIndex = (byte)datReader.ReadUInt16();
```

Comment:
Do you think it would be worth adding a "Read(DatReader datReader)" method to AnimationPartChange here?  Seems like it would make sense.